src/validation/category/edit.ts

```diff
@@ -0,0 +1,10 @@
+import * as yup from 'yup'
+
+const editTitleValidation = yup.object().shape({
+  title: yup.string().required().trim()
+})
+
+const editIDValidation = yup.object().shape({
+  categoryId: yup.number().required()
+})
```

Comment:
This is the same validation you used in the above file and the one below, you could use the same validator for all these controllers since you are only validating the `categoryId` which will be a positive number and is required.